Project/Galagram/ViewModel/CommandBase.cs

```diff
@@ -5,6 +5,11 @@
     /// </summary>
     public abstract class CommandBase : System.Windows.Input.ICommand
     {
+        /// <summary>
+        /// State of the command executing
+        /// </summary>
+        public Enums.Admin.CommandState CommandState { get; set; } = Enums.Admin.CommandState.Default;
```

Comment:
probably should be field, and initialized in constructor. Not goes to convention rule

Reply:
Where is such rule in convention?

Reply:
![image](https://user-images.githubusercontent.com/24938726/53158463-50666880-35cc-11e9-975a-af7c64e2a77c.png)

Reply:
Ok, you are right.